run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+cd /usr/src/app
+case ${DOCKER_STATE} in
+migrate)
+    echo "running migrate"
+    bundle exec rake db:migrate
+    ;;
+seed)
+    echo "running seed"
+    bundle exec rake db:migrate
+    bundle exec rake db:seed
+    ;;
+vagrant)
+    echo "running seed"
```

Comment:
I wonder whether it should have a more descriptive name than `vagrant`, which leads me to wonder why it's a separate step from `create`.